tests/templates/kuttl/smoke/30-assert.yaml.j2

```diff
@@ -2,6 +2,22 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestAssert
 timeout: 600
+commands:
```

Comment:
just one nit: if this is moved to a following step (31-assert), then the log output is much cleaner.

Reply:
Indeed. Done in https://github.com/stackabletech/hdfs-operator/pull/576/commits/e6bb0d24d3aff7575c09b7180698fcaeebd2c54b

Reply:
Thanks!